tutorials/arvo/claz.md

```diff
@@ -56,21 +52,21 @@ For this guide, we'll continue on the same ship.
 Run `|start %eth-sender` and then you can use it to sign the transaction file.
 
 ```
-:eth-sender [%sign %/transaction-signed/txt %/transaction/eth-txs %/mykey/txt ~[80]]
+:eth-sender [%sign %/spawn-signed/txt %/spawn/eth-txs %/mykey/txt ~[40 80]]
 ```
 
 Let's again break this down.
 
-`%sign` is the action we are going to perform here.
+`%sign` is the action we are going to perform here. `eth-sender` can also be used to submit batches of transactions to the network in various ways, but details on that are outside of the scope of this guide.
 
-`%/transaction-signed/txt` is the path to the output file.
+`%/spawn-signed/txt` is the path to the output file.
 
-`%/transaction/eth-txs` is the path to the transaction file we generated earlier. It will need to be accessible so don't forget to `|comit %home` if you moved it to a different pier.
+`%/spawn/eth-txs` is the path to the transaction file we generated earlier. It will need to be accessible so don't forget to `|comit %home` if you moved it to a different pier.
 
-`%/mykey/txt` is the path to your private key file. This file is most likely the reason you will want to use an airgapped machine so that your private key never touches a networked computer. Opsec is left to the reader.
+`%/mykey/txt` is the path to your private key file. It should contain your private key beginning with 0x and followed by a hex string with no dots where capital letters are fine. This file is most likely the reason you will want to use an airgapped machine so that your private key never touches a networked computer. Opsec is left to the reader.
```

Comment:
"no dots where" -> "no dots, and where", or maybe just remove the part about capital letters?